run/test/specs/onboarding_pp.spec.ts

```diff
@@ -0,0 +1,45 @@
+import { bothPlatformsIt } from '../../types/sessionIt';
+import { PrivacyPolicyButton, SplashScreenLinks } from './locators/onboarding';
+import { runOnlyOnAndroid, runOnlyOnIOS } from './utils';
+import { closeApp, openAppOnPlatformSingleDevice, SupportedPlatformsType } from './utils/open_app';
+
+// risk assessment for later: high, because of the legal nature of these terms 
+bothPlatformsIt('Onboarding privacy policy', onboardingPP)
+
+const ppURL = 'https://getsession.org/privacy-policy';
+let urlField; 
+
+async function onboardingPP(platform: SupportedPlatformsType) {
+    const { device } = await openAppOnPlatformSingleDevice(platform);
+    // Tap the text at the bottom of the splash screen to bring up the TOS/PP links modal
+    await device.clickOnElementAll(new SplashScreenLinks(device));
+    // Tap Privacy Policy
+    await device.clickOnElementAll(new PrivacyPolicyButton(device));
+    // Grabbing the URL from the browser works differently per platform 
+    if (platform === 'ios') {
+        // Tap the Safari navigation bar to reveal the URL 
+        await device.clickOnByAccessibilityID('TabBarItemTitle')
+        // Grab URL from the navigation bar
+        urlField = await device.waitForTextElementToBePresent({strategy: 'accessibility id', selector: 'URL'})
```

Comment:
This should be a new locator class

Reply:
I purposely did not record any native UI as locator classes - is that something we want to do?

Reply:
yes

Reply:
this has been addressed in the new `browsers.ts` locator file